pyopendoc/writer.py

```diff
@@ -75,6 +75,27 @@ def write_to_table(self, table_name, column, row, value, refresh=False, target=N
 
         span.text = value
 
+    def write_to_frame(self, frame_name, x_value, y_value, value=None, target=None, unit="in"):
+        """Writes to x, y coordinate of a specified frame.
+        Defaults unit to inches.
+
+        """
+        target_file = target if target else self.CONTENT_FILE
+        xml_file = self.get_file(target_file)
+        if xml_file.filetype != "XML":
+            raise TypeError("Not an XML file!")
+
+        for df in xml_file.root.findall(".//draw:frame", self.NAMESPACES):
+            frame = df.get("{%s}name" % self.NAMESPACES["draw"])
+            if frame == frame_name:
```

Comment:
Thanks - and yes just the one frame. I returned a string, but could I have returned the frame? I know it's bad form to return nothing in JS, is that the same case with Python?